src/dotnet-interactive-vscode/insiders/src/notebookControllers.ts

```diff
@@ -67,8 +67,8 @@ export class DotNetNotebookKernel {
             }
         });
         this.commonControllerInit(jupyterController);
-        this.disposables.push(vscode.notebook.onDidOpenNotebookDocument(async notebook => {
-            if (notebook.viewType === jupyterViewType && isDotNetNotebook(notebook)) {
+        this.disposables.push(vscode.notebooks.onDidOpenNotebookDocument(async notebook => {
```

Comment:
With today breaking changes this `vscode.notebooks.onDidOpenNotebookDocument` becomes `vscode.workspace.onDidOpenNotebookDocument`

Reply:
This namespace change isn't live yet, but I'll do it as soon as it comes down.